src/browser/components/SettingsPage.jsx

```diff
@@ -55,26 +47,6 @@ export default class SettingsPage extends React.Component {
         }
       });
     });
-
-    // when the config object changes here in the renderer process, tell the main process to reload its config object to get the changes
-    config.on('synchronize', () => {
-      ipcRenderer.send('reload-config');
-    });
-
-    // listen for any config reload requests from the main process to reload configuration changes here in the renderer process
-    ipcRenderer.on('reload-config', () => {
-      config.reload();
-    });
-
-    ipcRenderer.on('add-server', () => {
-      this.setState({
-        showAddTeamForm: true,
-      });
-    });
-
-    ipcRenderer.on('switch-tab', (event, key) => {
-      backToIndex(key);
-    });
```

Comment:
Already listening in MainPage.
Redundant now that `SettingsPage` is a modal, controlled in `MainPage`